db/index.js

```diff
@@ -1,13 +1,69 @@
 // Connect to DB
 const { Client } = require('pg');
-const DB_NAME = 'change-this-name'
+const DB_NAME = 'localhost:5432/graceshopper'
 const DB_URL = process.env.DATABASE_URL || `postgres://${ DB_NAME }`;
 const client = new Client(DB_URL);
 
 // database methods
+const createProduct = async (productField) => {
+  const { name, description, price, category } = productField;
```

Comment:
MILD - Create a "safe list" so that people can't attempt to insert a non-defined column. like `product.dog` which doesn't exist....

Reply:
Will handle this down the line ;)